frontend/src/store/course/actions.ts

```diff
@@ -118,24 +118,34 @@ const getActiveInterviewsCategoryIdsByUserId = createAsyncThunk<
 
 const updateIsMentorBecomingEnabled = createAsyncThunk<
   boolean,
-  void,
+  number,
   AsyncThunkConfig
->(ActionType.SET_IS_MENTOR_BECOMING_ENABLED, async (_, { extra, getState }) => {
-  const {
-    course: { course },
-  } = getState();
+>(
+  ActionType.SET_IS_MENTOR_BECOMING_ENABLED,
+  async (payload, { extra, getState }) => {
```

Comment:
```suggestion
  async (userId, { extra, getState }) => {
```
Maybe it'll be more clearer